components/testdata/reports/cloc/cloc-by-file.json

```diff
@@ -4586,11 +4586,6 @@
   "comment": 2,
   "code": 4,
   "language": "Python"},
-"components/ldap/Dockerfile" :{
-  "blank": 2,
-  "comment": 0,
-  "code": 4,
-  "language": "Dockerfile"},
```

Comment:
This is unrelated testdata. Removing these lines invalidates the total figures in this file. So this change should be reverted.